### PR TITLE
Absolute nested path workaround for local persistent volumes

### PR DESCRIPTION
**Pull request by sascala**
@jieyu , if you have anything to add to the new section, please comment!

### CODE REVIEW

**Comment by joel-hamill** [comment]
@sascala can this be merged?

File: 1.7/usage/storage/persistent-volume.md [hunk 1]
```diff
@@ -34,24 +34,54 @@ Configure a persistent volume with the following options:
 }
 ```
 
-- `containerPath`: The path where your application will read and write data. This must be a single-level path relative to the container; it cannot contain a forward slash (`/`). (`"data"`, but not `"/data"`, `"/var/data"` or `"var/data"`).
+- `containerPath`: The path where your application will read and write data. This must be a single-level path relative to the container; it cannot contain a forward slash (`/`). (`"data"`, but not `"/data"`, `"/var/data"` or `"var/data"`). If your application requires absolute nested paths, [use this configuration](#nested-paths).
```
Comment (jieyu) [inline 1.7/usage/storage/persistent-volume.md]: If your application requires absolute or nested paths?

**Comment by sascala** [comment]
@joel-hamill Jie had some comments I'd like to address before merging.

**Comment by sascala** [comment]
@joel-hamill you can merge

**Pull request merged**